docs/en/guides/security-alarm-system.md

```
@@ -79,11 +79,11 @@ The Security Alarm System Project utilizing Ultralytics YOLO11 integrates advanc
         cv2.destroyAllWindows()  # destroy all opened windows
         ```
 
-That's it! When you execute the code, you'll receive a single notification on your email if any object is detected. The notification is sent immediately, not repeatedly. However, feel free to customize the code to suit your project requirements.
+When you run the code, you will receive a single email notification if any object is detected. The notification is sent immediately, not repeatedly. You can customize the code to suit your project requirements.
 
 #### Email Received Sample
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)